hashring/rbtree.go

```diff
@@ -20,47 +20,35 @@
 
 package hashring
 
-// TODO: change RBNode.val from int to int64?
-
-// An RBTree is an automatically balancing tree of KVP pairs (presumably)
-type RBTree struct {
-	root *RBNode
+// RedBlackTree is an implemantation of a Red Black Tree
+type RedBlackTree struct {
+	root *Node
 	size int
 }
 
-// An RBNode is a node in the RBTree
-type RBNode struct {
+// Node is a node of the RedBlackTree
+type Node struct {
 	val   int
 	str   string
-	left  *RBNode
-	right *RBNode
+	left  *Node
+	right *Node
 	red   bool
 }
 
-// Size returns the number of nodes in the RBTree
-func (t *RBTree) Size() int {
+// Size returns the number of nodes in the RedBlackTree
+func (t *RedBlackTree) Size() int {
 	return t.size
 }
 
-// Child returns the left or right node of the RBTree
-func (n *RBNode) Child(right bool) *RBNode {
+// Child returns the left or right node of the RedBlackTree
+func (n *Node) Child(right bool) *Node {
```

Comment:
This package now exports a type called hashring.Node, which feels a little off to me (since it's not really a "HashRing node" so much as it is an implementation detail).  Should it be private to the package?